zipkin-server/src/main/java/zipkin/server/ZipkinServer.java

```diff
@@ -28,8 +45,23 @@
   }
 
   public static void main(String[] args) {
-    new SpringApplicationBuilder(ZipkinServer.class)
+    SpringApplicationBuilder builder = new SpringApplicationBuilder(ZipkinServer.class)
       .banner(new ZipkinBanner())
-      .properties("spring.config.name=zipkin-server").run(args);
+      .properties(
+        EnableAutoConfiguration.ENABLED_OVERRIDE_PROPERTY + "=false",
+        "spring.config.name=zipkin-server");
+
+    try {
+      Class<?> actuatorClass = Class.forName("zipkin.server.ActuatorConfiguration");
```

Comment:
the next step would be to switch the maven profile that excludes actuator to opt-out of compiling this